### PR TITLE
Table header freeze

### PR DESCRIPTION
**Pull request by jstirling91**
885734_1280x720

### CODE REVIEW

**Comment by crocodoyle** [comment]
Typo in image, should be *Tables*

**Comment by crocodoyle** [comment]
This PR improves all tables, where it should only apply to the top 1% of tables.

**Comment by alisterdev** [comment]
Not sure what it is supposed to do or how to test it... Good meme though!

File: htdocs/js/jquery.dynamictable.js [hunk 1]
```diff
@@ -130,6 +149,21 @@
       '<span class="glyphicon glyphicon-chevron-right"></span></a>'
     );
   };
+  var headerAlign = function(table, headers) {
+    var tableHeaders = $(table).find("thead").children().children();
+    var fixedHeaders = $(headers).find("thead").children().children();
```
Comment (alisterdev) [inline htdocs/js/jquery.dynamictable.js]: Suggest using this `$(headers).find("thead").children().children().not('.static-col');` to eliminate the need for the `if` in the loop

File: htdocs/js/jquery.dynamictable.js [hunk 2]
```diff
@@ -85,10 +85,17 @@
   var checkOverflow = function(wrapper, rightLink, leftLink, headCol) {
     var staticCol = (headCol !== undefined);
     var element = wrapper;
+    var table = wrapper.children[0];
```
Comment (alisterdev) [inline htdocs/js/jquery.dynamictable.js]: Would be nice to know what the magic numbers stand for

**Comment by xlecours** [comment]
@jstirling91 This is failing Travis. (eslint and integration tests)

**Comment by jacobpenny** [comment]
header-freeze-issue

It looks like there is an issue with this when used with the side-to-side scrollers. I end up getting an extra horizontal scroll bar. Other than that it works well.

**Comment by jstirling91** [comment]
@jacobpenny Which browser are you using?

**Comment by jacobpenny** [comment]
@jstirling91 Chrome Version 58.0.3029.110 (64-bit)

**Comment by jstirling91** [comment]
@jacobpenny I couldn't find a simple fix to remove the scroll bar completely but I removed the margin from the bottom so the scroll bar is within the header and not below

**Comment by jacobpenny** [comment]
Looks better. The scroll bar is still there (as you said), and you can scroll it so that it gets out of alignment with the the rest of the rows, but if you and @driusan are fine with that let's merge.

**Comment by driusan** [comment]
I'm willing to accept that to get this 6 month old PR merged.. but can you create a redmine ticket?

We'll decide while testing if it's a bug that needs to be fixed for this release or not, but it can go into another PR.

**Comment by johnsaigle** [comment]
@jacobpenny could you explicitly approve this so we could get it moving down the review pipeline?

**Pull request merged**